src/main/java/pl/envelo/moovelo/service/event/EventService.java

```diff
@@ -187,10 +185,66 @@ public Page<BasicUser> getUsersWithAccess(Long eventId, int page, int size) {
     }
 
     public static <T> Page<T> listToPage(final Pageable pageable, List<T> list) {
-        int first = Math.min(Long.valueOf(pageable.getOffset()).intValue(), list.size());;
+        int first = Math.min(Long.valueOf(pageable.getOffset()).intValue(), list.size());
         int last = Math.min(first + pageable.getPageSize(), list.size());
         return new PageImpl<>(list.subList(first, last), pageable, list.size());
     }
-}
 
+    @Transactional
+    public void setStatus(Long eventId, Long userId, String status) {
+        log.info("EventService - setStatus()");
+
+        Event event = getEventById(eventId);
+        BasicUser user = basicUserService.getBasicUserById(userId);
```

Comment:
tutaj warto sprawdzić, czy User ma dostęp do danego wydarzenia, tj. czyli czy zawiera się w liście event.usersWithAccess. Teoretycznie w przypadku Eventu ogólnego typu nie ma takiej potrzeby, ale jeśli zgodnie z koncepcją refactoru Ryśka będziemy wykorzystywać metody z eventService w odniesieniu do wydarzeń typu Internal, to takie zabezpieczenie się przyda

Reply:
@mstebel sprawdzenie dodane